### PR TITLE
Update Gree documentation add support for light panel switch

### PR DESCRIPTION
**Pull request by cmroche**
## Proposed change

Updating documentation, clean up and add information about support platforms, including a newly added switch to control the device light panel.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/42979
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/gree.markdown [hunk 1]
```diff
@@ -11,14 +11,53 @@ ha_codeowners:
 ha_domain: gree
 ---
 
-The Gree integration allows you to control a [Gree Smart HVAC](http://global.gree.com/).
+The Gree integration allows you to control a [Gree Smart HVAC](http://global.gree.com/) in Home Assistant.
+
+There is currently support for the following device types within Home Assistant:
+
+- [Configuration](#configuration)
+- [Supported models](#supported-models)
```
Comment (klaasnicolaas) [inline source/_integrations/gree.markdown]: these are no platform types 😄

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/gree.markdown (hunk 1 above)
Comment (cmroche) [reply source/_integrations/gree.markdown]: VS Code markdown plugin is mangling this on save. Anyway, fixed now.

File: source/_integrations/gree.markdown [hunk 2]
```diff
@@ -11,14 +11,51 @@ ha_codeowners:
...
+## Climate
+
+The `gree` climate platform integrates Gree HVAC systems into Home Assistant, enabling control of setting the following parameters:
+
+- [**set_hvac_mode**](/integrations/climate/#service-climateset_hvac_mode)
+- [**target temperature**](/integrations/climate#service-climateset_temperature)
+- [**turn on/off**](/integrations/climate#service-climateturn_on)
+- [**fan mode**](/integrations/climate#service-climateset_fan_mode)
+- [**swing mode**](/integrations/climate#service-climateset_swing_mode)
+- [**set_preset_mode**](/integrations/climate#service-climateset_preset_mode)
```
Comment (frenck) [inline source/_integrations/gree.markdown]: These are service names, referencing to code, and should be escaped using backticks, not emphasized as bold text.

File: source/_integrations/gree.markdown [hunk 3]
```diff
@@ -11,14 +11,51 @@ ha_codeowners:
...
 
 Go to the integrations page in your configuration and click on **new integration** -> **Gree**.
 
 Gree Smart compatible devices are detected and added automatically.
 
+<div class='note'>
+
+Devices must first be bound to WiFi using the Gree+ app, or app provided by the device manufacturer. It is not possible to connect devices to WiFi using this integration at the current time.
+
+</div>
```
Comment (frenck) [inline source/_integrations/gree.markdown]:
```suggestion
<div class='note'>
Devices must first be bound to Wi-Fi using the Gree+ app, or app provided by the device manufacturer. It is not possible to connect devices to Wi-Fi using this integration at the current time.
</div>
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Pull request merged**